spec/javascripts/components/escape-link.spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env jasmine, jquery,  EscapeLink, */
+
+describe('Escape link component', function () {
+  'use strict'
+
+  var container
+  var escapeLinkElement
+  var escapeLinkModule
+
+  beforeEach(function () {
+    container = document.createElement('div')
+    container.innerHTML = '<a class="app-c-escape-link govuk-link" rel="nofollow noreferrer noopener" target="_blank" data-module="app-escape-link" href="https://www.gov.uk/">Leave this site</a>'
+    document.body.appendChild(container)
+    escapeLinkElement = document.querySelector('[data-module="app-escape-link"]')
+    escapeLinkModule = new EscapeLink(escapeLinkElement)
+    escapeLinkModule.replaceCurrentPage = function () {}
```

Comment:
Jasmine crashes on any `window.location.replace` calls, so I had to mock this in tests to avoid it.